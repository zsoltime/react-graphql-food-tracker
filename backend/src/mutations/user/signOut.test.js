'use strict';

const signOut = require('./signOut');

describe('signOut mutation', () => {
  it('clears the token cookie and returns a message', () => {
    const mockedContext = {
      response: {
        clearCookie: jest.fn(),
      },
    };

    const res = signOut(null, null, mockedContext);

    expect(mockedContext.response.clearCookie).toHaveBeenCalledWith(
      'token'
    );
    expect(res).toMatchSnapshot();
  });
});
