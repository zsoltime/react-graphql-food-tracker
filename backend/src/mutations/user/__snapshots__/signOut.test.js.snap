// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`signOut mutation clears the token cookie and returns a message 1`] = `
Object {
  "message": "Successfully logged out",
}
`;
